include/mcmini/mcmini_private.h

```diff
@@ -209,6 +209,14 @@ extern MCDeferred<MCStack> programState;
  */
 void mc_create_global_state_object();
 
+/**
+ * @brief Return id for the operation type of objectID
+ *
+ * This counts how many objects, i,  of the same type
+ * as objectId exist, for i < ojbectId.
+ */
+int visibleOperationId(int objectId);
```

Comment:
We should just change the naming for this. Your documentation states 

> This counts how many objects, i,  of the same type as objectId exist, for i < objectId.

i.e. it talks about visible _objects_, not visible _operations_. Perhaps `visibleObjectsOfType(objectId)` or something